fink_filters/filter_anomaly_notification/filter_utils.py

```diff
@@ -426,16 +426,22 @@ def get_cutout(ztf_id):
             cutout image in png format
 
     '''
+    # transfer cutout data
     r = requests.post(
-        'https://fink-portal.org/api/v1/cutouts',
-        json={
-            'objectId': ztf_id,
-            'kind': 'Science'
-        },
-        timeout=25
+        "https://fink-portal.org/api/v1/objects",
+        json={"objectId": ztf_id, "withcutouts": "True"},
     )
-    status_check(r, 'get cutouts')
-    return io.BytesIO(r.content)
+    if not status_check(r, 'get cutouts'):
+        return io.BytesIO()
+    # Format output in a DataFrame
+    pdf = pd.read_json(io.BytesIO(r.content))
+    data = np.log(np.array(pdf['b:cutoutScience_stampData'].to_numpy()[0], dtype=float))
```

Comment:
You do not need pandas here -- I suggest:

```diff
- pdf = pd.read_json(io.BytesIO(r.content))
- data = np.log(np.array(pdf['b:cutoutScience_stampData'].to_numpy()[0], dtype=float))
+ data = np.log(np.array(r.json()[0]['b:cutoutScience_stampData'], dtype=float))
```

Reply:
I fixed it, thanks!

Reply:
Thanks!